### ISSUE TITLE
How to introduce GANS and feature loss in transfer learning?

### ISSUE THREAD

**Issue by Sarah335** [idealo/image-super-resolution#216]
I'm looking to fine-tune the noise-cancel weights with MRI brain images and I'm not sure if I should start with only the PSNR loss (e.g. generator loss weight = 1.0, feature extractor and discriminator loss weights = 0), or if I could go straight to using the GANS and feature loss (e.g. generator loss weight = 0, feature extractor loss = 0.0833, discriminator loss = 0.01). 

Given the noise-cancel RDN was trained WITH adversarial and feature losses, am I safe to use these from the start?!  Curious to hear people's opinions/suggestions!
